src/Sidekick/Windows/Prediction/PredictionController.cs

```diff
@@ -1,41 +1,41 @@
 using System.Windows.Forms;
 using System.Windows.Input;
 using Sidekick.Business.Apis.PoePriceInfo.Models;
+using Sidekick.Core.Natives;
 using Cursor = System.Windows.Forms.Cursor;
 
 namespace Sidekick.Windows.Prediction
 {
-    public static class PredictionController
+    public class PredictionController
```

Comment:
This is a legacy view, so you do not need to keep it up to date. We are going to redo this in the future.